add(platform): limited and early availability tags (#29)
aiven/aiven-docs: docs/platform/howto/use-azure-privatelink.md

## Prerequisites

> :::important
> Azure Private Link is a
> [limited availability feature](/docs/platform/concepts/beta_services), which you can request from the sales team
> ([sales@aiven.io](mailto:sales@aiven.io)) or your account manager. During the limited
> availability stage, you can use the feature at no cost. If you want to
> continue using Azure Private Link after it reaches general availability,
> you\'ll be billed according to the latest applicable price.
> :::
> 

Reviewer: You used a [normal paragraph in this other page](https://github.com/aiven/aiven-docs/pull/29/files#diff-6551e4d317aa99a86c68a77d1d5cf9747e20bbcfbb2d9f7d20f91ce400de248aR10), let's do the same for simplicity and coherence?

```suggestion
-   This feature is in limited availability. Contact the sales team(mailto:sales@aiven.io) to enable it.
```
